verifier/trustpolicy/trustpolicy_test.go

```diff
@@ -518,7 +519,7 @@ func TestApplicableTrustPolicy(t *testing.T) {
 
 	// non-existing Registry Scope
 	policy, err = (&policyDoc).GetApplicableTrustPolicy("non.existing.scope/repo@sha256:hash")
-	if policy != nil || err == nil || err.Error() != "artifact \"non.existing.scope/repo@sha256:hash\" has no applicable trust policy" {
+	if policy != nil || err == nil || err.Error() != "artifact \"non.existing.scope/repo@sha256:hash\" has no applicable trust policy. Trust policy applicability for a given artifact is determined by registryScopes. To create Notation trust policy: https://notaryproject.dev/docs/quickstart/#create-a-trust-policy" {
```

Comment:
To create a trust policy, see: https://notaryproject.dev/docs/quickstart/#create-a-trust-policy.